cumulus/parachains/common/src/westend.rs

```diff
@@ -13,6 +13,24 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/// Universally recognized accounts.
+pub mod account {
```

Comment:
it would be good to have a different common for testnets since now this is published to crates.io and used for the pod runtimes. but this does not concern this PR